src/mixins/button.scss

```diff
@@ -0,0 +1,24 @@
+@mixin button {
```

Comment:
In the future I think we should probably break out a `Button` component rather than just having a sass mixin, but I agree this is a sensible migration path.